src/playwrightTest.ts

```diff
@@ -64,7 +64,7 @@ export class PlaywrightTest {
         'try { const pwtIndex = require.resolve("@playwright/test"); const version = require("@playwright/test/package.json").version; console.log(JSON.stringify({ pwtIndex, version})); } catch { console.log("undefined"); }',
       ], path.dirname(configFilePath));
       const { pwtIndex, version } = JSON.parse(pwtInfo);
-      const v = parseFloat(version);
+      const v = parseFloat(version.replace(/-(next|beta)$/, ''));
```

Comment:
this makes it work when we dogfood our repo, otherwise parseFloat didn't parse our version number in the package.json correctly.